src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/containerTerminated.groovy

```diff
@@ -0,0 +1,22 @@
+podTemplate(yaml:'''
+spec:
+  containers:
+  - name: jnlp
+    image: jenkins/inbound-agent:4.3-4 # alpine flavor has alpine 3.9 which doesn't have stress-ng
+    securityContext:
+      runAsUser: 0
+      privileged: true
+    resources:
+      limits:
+        memory: "256Mi"
+      requests:
+        memory: "256Mi"
+''') {
+  node(POD_LABEL) {
+    sh '''
+        apt-get update
+        apt-get install -y stress-ng
```

Comment:
Maybe better to just publish an image (e.g., Docker Hub automated build) with this preinstalled? Or kill a different container using an image that already has this tool installed? Or perhaps there is some stock Linux command which can be forced to do giant allocations with the right arguments?

Reply:
Yes, didn't want to go through the route of publishing a new docker image just for that purpose. I know @Dohbedoh had one set up but ideally I'd avoid depending on third party.